sharktank/tests/layers/sharded_conv2d_with_iree_test.py

```diff
@@ -75,12 +75,9 @@ def run_iree_module(devices: list[iree.runtime.HalDevice]):
         # TODO: make IREE able to load the parameters from the top parameter file
         # without having to specify the parameter file for each shard separately.
         parameter_index = iree.runtime.ParameterIndex()
-        for i in range(shard_count):
-            parameter_index.load(
-                file_path=str(
-                    Path(params_path).with_suffix(f".rank{i}{params_path.suffix}")
-                )
-            )
+        handles = get_file_handle(shard_count=shard_count, weight_path=params_path)
+        for handle in handles:
+            parameter_index.load_from_file_handle(handle, "irpa")
```

Comment:
This test was written before `load_iree_module`, but it would be nice to use it.